lib/ui/myplot/my_plot_detail_screen.dart

```diff
@@ -1,4 +1,5 @@
 import 'package:farmsmart_flutter/data/model/crop_entity.dart';
+import 'package:farmsmart_flutter/farmsmart_localizations.dart';
```

Comment:
This is being refactored in another branch, could we remove these changes as they will conflict and need to be changed.

Reply:
sure thing